### ISSUE TITLE
Cross signing

### ISSUE THREAD

**Issue by toger5** [matrix-org/pantalaimon#50]
is cross signing supported?

**Comment by poljar** [comment]
Not yet.

**Comment by toger5** [comment]
Is it now?
Or is there a way that i can veryfy all devices taht are unverified. so i can check in element if all of them are okay and than i can veryfy them all at once in pentalaimon

**Comment by poljar** [comment]
No, the client lib that pan uses is going through a rewrite of the crypto parts of the code, so it's taking a bit longer. 

You can manually verify devices with `panctl`, you could probably pass in commands that get executed with a shell if you want to do that in bulk:

```bash
echo "verify-device @example:localhost @example2:localhost DEVICEID" | panctl
```

**Comment by DamienCassou** [comment]
I get "pantalaimon => not trusted" in Element. Is that related to this issue? I don't understand how the command above could help to trust the pantalaimon "device" itself.

**Comment by poljar** [comment]
It would not help, you can:

1. Interactively verify Pantalaimon with Element, other devices will see Pantalaimon as verified but Pantalaimon will only see Element as verified.
2. Manually verify Pantalaimon with Element, other devices will see Pantalaimon as verified but Pantalaimon will see no devices as verified.
3. Manually verify another device with Pantalaimon, only Pantalaimon will see the other device as verified.

**Comment by Thaodan** [comment]
Related to matrix-nio/matrix-nio#229
